pavelib/quality.py

```diff
@@ -234,7 +241,7 @@ def run_complexity():
     Uses radon to examine cyclomatic complexity.
     For additional details on radon, see http://radon.readthedocs.org/
     """
-    system_string = 'cms/ lms/ common/ openedx/'
+    system_string = join(ALL_SYSTEMS, '/ ') + '/'
```

Comment:
This is a hold-over from my first attempt where I introduced a new top-level folder called "features". I could remove it from here, but it seemed like a useful generalization of the code rather than duplicating the systems in two different places.